src/transformers/models/glm4v/video_processing_glm4v.py

```diff
@@ -133,22 +143,18 @@ def sample_frames(
         if len(uniq) & 1:
             uniq.append(uniq[-1])
 
-        frame_indices = uniq
-        sampled_video = video[frame_indices]
-        full_second_idxs = [int(idx / video_fps) for idx in frame_indices]
-        second_idxs = full_second_idxs[::2]  # mrope
-        return sampled_video, second_idxs
+        return np.array(uniq)
 
     def _preprocess(
         self,
         videos: list[torch.Tensor],
-        video_metadata: Optional[Union[list[VideoMetadata], list[dict]]] = None,
+        do_convert_rgb: bool = True,
         do_resize: bool = True,
+        size: SizeDict = None,
```

Comment:
Any argument that defaults to `None` should be annotated as Optional

```suggestion
        size: Optional[SizeDict] = None,
```